### ISSUE TITLE
linux install error in sqlite dev dependency not installed

### ISSUE THREAD

**Issue by maks** [arxanas/git-branchless#17]
running: `cargo install git-branchless` gives:

```
[snip...]

stlib/x86_64-unknown-linux-gnu/lib/libcore-1395c6db3d116086.rlib" "-Wl,--end-group" "/home/maks/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcompiler_builtins-b4bd87926720b651.rlib" "-Wl,-Bdynamic" "-lsqlite3" "-lncursesw" "-ltinfo" "-lz" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc"
  = note: /usr/bin/ld: cannot find -lsqlite3
          collect2: error: ld returned 1 exit status
          

error: aborting due to previous error

error: failed to compile `git-branchless v0.3.1`, intermediate artifacts can be found at `/tmp/cargo-installNtxzqk`

Caused by:
  could not compile `git-branchless`
```
On Ubuntu, it was enough to use: `sudo apt install libsqlite3-dev` to install the required sqlite dependency to then successfully install git-branchless. 
Perhaps `cargo` has some way of checking for the native sqlite lib to be installed? Ortherwise might be worth documenting this dependency for installs via cargo.

**Comment by arxanas** [comment]
@maks Thanks for reporting. This might be fixed as of #13, since it bundles SQLite. Can you try installing the latest code (`cargo install --git https://github.com/arxanas/git-branchless`) and see if it builds without the SQLite dependency?

**Comment by maks** [comment]
Thanks for the quick response and yes that does fix it, I removed my system install of the sqlite dev package, uninstalled branchless, ran the cargo install again it failed, tried the git install and it worked fine.

**Comment by mslinn** [comment]
I have a similar problem, running Ubunto on WSL2:
```
$ cargo install git-branchless
    Updating crates.io index
  Installing git-branchless v0.3.1
   Compiling libc v0.2.97
   Compiling proc-macro2 v1.0.27
   Compiling autocfg v1.0.1
   Compiling cfg-if v1.0.0
   Compiling syn v1.0.73
   Compiling unicode-xid v0.2.2
   Compiling pkg-config v0.3.19
   Compiling version_check v0.9.3
   Compiling lazy_static v1.4.0
   Compiling log v0.4.14
   Compiling bitflags v1.2.1
   Compiling memchr v2.4.0
   Compiling strsim v0.9.3
   Compiling ident_case v1.0.1
   Compiling fnv v1.0.7
   Compiling unicode-width v0.1.8
   Compiling matches v0.1.8
   Compiling smallvec v1.6.1
   Compiling crossbeam-utils v0.8.5
   Compiling tinyvec_macros v0.1.0
   Compiling scopeguard v1.1.0
   Compiling percent-encoding v2.1.0
   Compiling ahash v0.4.7
   Compiling stable_deref_trait v1.2.0
   Compiling signal-hook v0.3.9
   Compiling doc-comment v0.3.3
   Compiling ppv-lite86 v0.2.10
   Compiling unicode-segmentation v1.7.1
   Compiling predicates-core v1.0.2
   Compiling array-macro v1.0.5
   Compiling once_cell v1.8.0
   Compiling regex-syntax v0.6.25
   Compiling strsim v0.8.0
   Compiling difference v2.0.0
   Compiling vec_map v0.8.2
   Compiling treeline v0.1.0
   Compiling xi-unicode v0.3.0
   Compiling anyhow v1.0.41
   Compiling ansi_term v0.11.0
   Compiling regex-automata v0.1.10
   Compiling fallible-streaming-iterator v0.1.9
   Compiling remove_dir_all v0.5.3
   Compiling maplit v1.0.2
   Compiling fallible-iterator v0.2.0
   Compiling instant v0.1.9
   Compiling proc-macro-error-attr v1.0.4
   Compiling ahash v0.6.3
   Compiling proc-macro-error v1.0.4
   Compiling num-traits v0.2.14
   Compiling num-integer v0.1.44
   Compiling num-iter v0.1.42
   Compiling num-rational v0.3.2
   Compiling textwrap v0.11.0
   Compiling unicode-bidi v0.3.5
   Compiling tinyvec v1.2.0
   Compiling lock_api v0.4.4
   Compiling libsqlite3-sys v0.20.1
   Compiling form_urlencoded v1.0.1
   Compiling owning_ref v0.4.1
   Compiling hashbrown v0.9.1
   Compiling heck v0.3.3
   Compiling predicates v1.0.8
   Compiling predicates-tree v1.0.2
   Compiling hashlink v0.6.0
   Compiling unicode-normalization v0.1.19
   Compiling quote v1.0.9
   Compiling getrandom v0.2.3
   Compiling signal-hook-registry v1.4.0
   Compiling atty v0.2.14
   Compiling time v0.1.43
   Compiling mio v0.7.13
   Compiling parking_lot_core v0.8.3
   Compiling term_size v0.3.2
   Compiling terminal_size v0.1.17
   Compiling wait-timeout v0.2.0
   Compiling jobserver v0.1.22
   Compiling aho-corasick v0.7.18
   Compiling bstr v0.2.16
   Compiling crossbeam-channel v0.5.1
   Compiling rand_core v0.6.3
   Compiling clap v2.33.3
   Compiling colored v1.9.3
   Compiling idna v0.2.3
   Compiling signal-hook v0.1.17
   Compiling parking_lot v0.11.1
   Compiling rusqlite v0.24.2
   Compiling cc v1.0.68
   Compiling num-complex v0.3.1
   Compiling rand_chacha v0.3.1
   Compiling assert_cmd v1.0.5
   Compiling regex v1.5.4
   Compiling url v2.2.2
   Compiling crossterm v0.19.0
   Compiling darling_core v0.10.2
   Compiling chrono v0.4.19
   Compiling rand v0.8.4
   Compiling libz-sys v1.1.3
   Compiling libgit2-sys v0.12.21+1.1.0
   Compiling ncurses v5.101.0
   Compiling console v0.14.1
   Compiling enum-map-derive v0.4.6
   Compiling fn-error-context v0.1.2
   Compiling structopt-derive v0.4.14
   Compiling num v0.3.1
   Compiling simple_logger v1.11.0
   Compiling tempfile v3.2.0
   Compiling darling_macro v0.10.2
   Compiling enum-map v0.6.4
   Compiling structopt v0.3.21
   Compiling darling v0.10.2
   Compiling wasmer_enumset_derive v0.5.0
   Compiling wasmer_enumset v1.0.1
   Compiling cursive_core v0.2.2
   Compiling cursive v0.16.3
   Compiling git2 v0.13.20
   Compiling git-branchless v0.3.1
error: linking with `cc` failed: exit status: 1
  |
  = note: "cc" "-m64" "-Wl,--eh-frame-hdr" "-Wl,-znoexecstack" "-Wl,--as-needed" "-L" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.0.rcgu.o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.1.rcgu.o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.10.rcgu.o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.11.rcgu.o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.12.rcgu.o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.13.rcgu.o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.14.rcgu.o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.15.rcgu.o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.2.rcgu.o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.3.rcgu.o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.4.rcgu.o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.5.rcgu.o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.6.rcgu.o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.7.rcgu.o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.8.rcgu.o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.git_branchless.ddhhecow-cgu.9.rcgu.o" "-o" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c" "/tmp/cargo-installL9yEEa/release/deps/git_branchless-bc194cd28e85a30c.1gojte2nyt281zqi.rcgu.o" "-Wl,--gc-sections" "-pie" "-Wl,-zrelro" "-Wl,-znow" "-Wl,-O1" "-nodefaultlibs" "-L" "/tmp/cargo-installL9yEEa/release/deps" "-L" "/tmp/cargo-installL9yEEa/release/build/libgit2-sys-bf4eae7891533b59/out/build" "-L" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "-Wl,-Bstatic" "/tmp/cargo-installL9yEEa/release/deps/libstructopt-864788ddde09ce50.rlib" "/tmp/cargo-installL9yEEa/release/deps/libclap-4abd01a383152dfe.rlib" "/tmp/cargo-installL9yEEa/release/deps/libvec_map-5a633ecabcbede8a.rlib" "/tmp/cargo-installL9yEEa/release/deps/libtextwrap-e116821781120f98.rlib" "/tmp/cargo-installL9yEEa/release/deps/libstrsim-1f1955367fd625a1.rlib" "/tmp/cargo-installL9yEEa/release/deps/libansi_term-6a98a04d1085fffe.rlib" "/tmp/cargo-installL9yEEa/release/deps/libsimple_logger-f0e1212beedbb7ad.rlib" "/tmp/cargo-installL9yEEa/release/deps/libcolored-c25ed84943855cc5.rlib" "/tmp/cargo-installL9yEEa/release/deps/libatty-0769eadf31f6a523.rlib" "/tmp/cargo-installL9yEEa/release/deps/libbranchless-5490a1848622f848.rlib" "/tmp/cargo-installL9yEEa/release/deps/libtempfile-f511d8c60df99d8e.rlib" "/tmp/cargo-installL9yEEa/release/deps/librand-7ce948d52eab7af9.rlib" "/tmp/cargo-installL9yEEa/release/deps/librand_chacha-5ea7180e2b771da3.rlib" "/tmp/cargo-installL9yEEa/release/deps/libppv_lite86-fe49843071894a3d.rlib" "/tmp/cargo-installL9yEEa/release/deps/librand_core-f73ee491b9bd2d01.rlib" "/tmp/cargo-installL9yEEa/release/deps/libremove_dir_all-b2850f33bbe50cfc.rlib" "/tmp/cargo-installL9yEEa/release/deps/libassert_cmd-0579a514e1915d9e.rlib" "/tmp/cargo-installL9yEEa/release/deps/libwait_timeout-5b2ab81b6c9247dd.rlib" "/tmp/cargo-installL9yEEa/release/deps/libdoc_comment-1f4ac5171520de65.rlib" "/tmp/cargo-installL9yEEa/release/deps/libbstr-b7964b481851fb7b.rlib" "/tmp/cargo-installL9yEEa/release/deps/libregex_automata-9610ed59622be30a.rlib" "/tmp/cargo-installL9yEEa/release/deps/libpredicates_tree-ce678eb71a3ad026.rlib" "/tmp/cargo-installL9yEEa/release/deps/libtreeline-34b9c73d50b22cfc.rlib" "/tmp/cargo-installL9yEEa/release/deps/libpredicates-7129438196ed45ce.rlib" "/tmp/cargo-installL9yEEa/release/deps/libdifference-2b54644cc34f2458.rlib" "/tmp/cargo-installL9yEEa/release/deps/libpredicates_core-e8a35e62154119d4.rlib" "/tmp/cargo-installL9yEEa/release/deps/librusqlite-93f32d8561ee293e.rlib" "/tmp/cargo-installL9yEEa/release/deps/libfallible_streaming_iterator-256a346856684565.rlib" "/tmp/cargo-installL9yEEa/release/deps/libfallible_iterator-0d749ef4ce014b3d.rlib" "/tmp/cargo-installL9yEEa/release/deps/libhashlink-1ffe7df39880ff6f.rlib" "/tmp/cargo-installL9yEEa/release/deps/libhashbrown-9ac4bf182dc51594.rlib" "/tmp/cargo-installL9yEEa/release/deps/libahash-64ec9fee67e7ec05.rlib" "/tmp/cargo-installL9yEEa/release/deps/liblibsqlite3_sys-b9efa51e00afc1cb.rlib" "/tmp/cargo-installL9yEEa/release/deps/libcursive-6a61b2aaf30a7a42.rlib" "/tmp/cargo-installL9yEEa/release/deps/libncurses-c7fa15d76dc73d12.rlib" "/tmp/cargo-installL9yEEa/release/deps/libmaplit-1cbd2da4dbbd610a.rlib" "/tmp/cargo-installL9yEEa/release/deps/libcrossterm-bc01f0097c079198.rlib" "/tmp/cargo-installL9yEEa/release/deps/libsignal_hook-430fefc3afeb6a7e.rlib" "/tmp/cargo-installL9yEEa/release/deps/libmio-5e7e75d5ad14933e.rlib" "/tmp/cargo-installL9yEEa/release/deps/libparking_lot-56ecba444c4c0626.rlib" "/tmp/cargo-installL9yEEa/release/deps/libparking_lot_core-f03ed7342a032298.rlib" "/tmp/cargo-installL9yEEa/release/deps/libsmallvec-6afca1b3452e7641.rlib" "/tmp/cargo-installL9yEEa/release/deps/liblock_api-e5bdbca3009eddde.rlib" "/tmp/cargo-installL9yEEa/release/deps/libscopeguard-15331d4a8e3b48c1.rlib" "/tmp/cargo-installL9yEEa/release/deps/libinstant-4413f3191560175c.rlib" "/tmp/cargo-installL9yEEa/release/deps/libsignal_hook-41d0f1211c7d7b96.rlib" "/tmp/cargo-installL9yEEa/release/deps/libsignal_hook_registry-049c141211a53453.rlib" "/tmp/cargo-installL9yEEa/release/deps/libcursive_core-43bfbefd7d17ceca.rlib" "/tmp/cargo-installL9yEEa/release/deps/libahash-2572e9bfbe16f410.rlib" "/tmp/cargo-installL9yEEa/release/deps/libgetrandom-b4b8e26e7a755005.rlib" "/tmp/cargo-installL9yEEa/release/deps/libonce_cell-0cb5b3556d9b9aca.rlib" "/tmp/cargo-installL9yEEa/release/deps/libchrono-7d7ae39f52ef7c5a.rlib" "/tmp/cargo-installL9yEEa/release/deps/libtime-f915c0c2fad295b6.rlib" "/tmp/cargo-installL9yEEa/release/deps/libcrossbeam_channel-d042479ac865a356.rlib" "/tmp/cargo-installL9yEEa/release/deps/libcrossbeam_utils-bcafed872dfeba0b.rlib" "/tmp/cargo-installL9yEEa/release/deps/libnum-13b65c1e52e37f57.rlib" "/tmp/cargo-installL9yEEa/release/deps/libnum_iter-b20dcdfbd3565974.rlib" "/tmp/cargo-installL9yEEa/release/deps/libnum_rational-7d43f20990596682.rlib" "/tmp/cargo-installL9yEEa/release/deps/libnum_integer-48e4203f7359201a.rlib" "/tmp/cargo-installL9yEEa/release/deps/libnum_complex-cf10660d2edb8347.rlib" "/tmp/cargo-installL9yEEa/release/deps/libenum_map-7388bf8a4daa5eca.rlib" "/tmp/cargo-installL9yEEa/release/deps/libarray_macro-86703fd63cfa6019.rlib" "/tmp/cargo-installL9yEEa/release/deps/libowning_ref-e40b0f5faf9206d8.rlib" "/tmp/cargo-installL9yEEa/release/deps/libstable_deref_trait-fcd3972d38a9d4fa.rlib" "/tmp/cargo-installL9yEEa/release/deps/libxi_unicode-bf53d2f146648c27.rlib" "/tmp/cargo-installL9yEEa/release/deps/libunicode_segmentation-c68b81dfa41880ad.rlib" "/tmp/cargo-installL9yEEa/release/deps/libwasmer_enumset-19601d952fc9ba13.rlib" "/tmp/cargo-installL9yEEa/release/deps/libnum_traits-0d78845baf222332.rlib" "/tmp/cargo-installL9yEEa/release/deps/libconsole-e43224eeb395a65d.rlib" "/tmp/cargo-installL9yEEa/release/deps/libterminal_size-1db04913c8ba1a1b.rlib" "/tmp/cargo-installL9yEEa/release/deps/libregex-d77a181d4ffe1a8f.rlib" "/tmp/cargo-installL9yEEa/release/deps/libaho_corasick-87366150b7da3371.rlib" "/tmp/cargo-installL9yEEa/release/deps/libmemchr-33700248bab0e8e8.rlib" "/tmp/cargo-installL9yEEa/release/deps/libregex_syntax-3686789fe085b6b6.rlib" "/tmp/cargo-installL9yEEa/release/deps/libunicode_width-b995ec368e728108.rlib" "/tmp/cargo-installL9yEEa/release/deps/liblazy_static-b29a250468ee713f.rlib" "/tmp/cargo-installL9yEEa/release/deps/libgit2-76442e90a0dcda61.rlib" "/tmp/cargo-installL9yEEa/release/deps/liburl-3cb2757e7b97b60f.rlib" "/tmp/cargo-installL9yEEa/release/deps/libidna-c41b5b8397faa01b.rlib" "/tmp/cargo-installL9yEEa/release/deps/libunicode_normalization-82c7e2cd267b7f43.rlib" "/tmp/cargo-installL9yEEa/release/deps/libtinyvec-3cd7e5cf59b08923.rlib" "/tmp/cargo-installL9yEEa/release/deps/libtinyvec_macros-6722b761eaeeeaf5.rlib" "/tmp/cargo-installL9yEEa/release/deps/libunicode_bidi-0d250cc156352038.rlib" "/tmp/cargo-installL9yEEa/release/deps/libform_urlencoded-276c95e762addf81.rlib" "/tmp/cargo-installL9yEEa/release/deps/libpercent_encoding-44320724dae1ad17.rlib" "/tmp/cargo-installL9yEEa/release/deps/libmatches-5569a903734de6f8.rlib" "/tmp/cargo-installL9yEEa/release/deps/liblog-14fc5f636a860f6b.rlib" "/tmp/cargo-installL9yEEa/release/deps/libcfg_if-6399ff0a7b04fb24.rlib" "/tmp/cargo-installL9yEEa/release/deps/liblibgit2_sys-2b9d0dbd99e8cfab.rlib" "/tmp/cargo-installL9yEEa/release/deps/liblibz_sys-a05c9d0972e1ec64.rlib" "/tmp/cargo-installL9yEEa/release/deps/liblibc-25183bb6867e795a.rlib" "/tmp/cargo-installL9yEEa/release/deps/libbitflags-61ee16c26132f2e5.rlib" "/tmp/cargo-installL9yEEa/release/deps/libanyhow-6473368ee4285359.rlib" "-Wl,--start-group" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd-b6b48477bfa8c673.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libpanic_unwind-f560ec02638f7ffe.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libminiz_oxide-9c8eadb7013c9e0b.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libadler-8b0ec8dbdb85d0bf.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libobject-ba5d5ee707c805d2.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libaddr2line-55166126dbdd5e46.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libgimli-c327b365eae3b2f3.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libstd_detect-416439b546a0d033.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_demangle-2581188d29552e15.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libhashbrown-da7b2635bfcce6ef.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_std_workspace_alloc-09200ed1945e7b2b.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libunwind-223ac369b29f5000.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcfg_if-39562fe6600dd936.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liblibc-9b411bb7a19f81b3.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/liballoc-64ea0581d80339f7.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/librustc_std_workspace_core-b2dbda88b377d685.rlib" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcore-2a8415a96ed1d7dc.rlib" "-Wl,--end-group" "/home/mslinn/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib/libcompiler_builtins-c4d9a5b072ee3191.rlib" "-Wl,-Bdynamic" "-lsqlite3" "-lncursesw" "-ltinfo" "-lz" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc"
  = note: /bin/ld: cannot find -lsqlite3
          collect2: error: ld returned 1 exit status


error: aborting due to previous error

error: failed to compile `git-branchless v0.3.1`, intermediate artifacts can be found at `/tmp/cargo-installL9yEEa`

Caused by:
  could not compile `git-branchless`

To learn more, run the command again with --verbose.
```

**Comment by arxanas** [comment]
Released v0.3.2, which should fix this issue.
